src/style.css

```diff
@@ -0,0 +1,132 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.firstSearch {
+  margin: 20px 0;
+}
+
+.firstInput {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  height: 70px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  font-weight: normal;
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-left: 62px;
+}
+
+.firstInput::placeholder {
+  font-weight: normal;
+  font-family: inherit;
+  font-size: 16px;
+}
```

Comment:
Will be better to add one main class for inputs and use it for setting styles to placeholder